### PR TITLE
Updated Czech translation

### PR DESCRIPTION
**Pull request by jan-vince**
Added missing translations

### CODE REVIEW

File: modules/backend/lang/cs/lang.php [hunk 1]
```diff
@@ -53,7 +60,8 @@
         'menu_label' => 'Plocha',
         'widget_label' => 'Widget',
         'widget_width' => 'Šířka',
-        'full_width' => 'plná šířka',
+        'full_width' => 'Plná šířka',
+        'manage_widgets' => 'Manage widgets',
```
Comment (chesio) [inline modules/backend/lang/cs/lang.php]: Not actually translated ;)

File: modules/backend/lang/cs/lang.php [hunk 2]
```diff
@@ -93,24 +130,44 @@
...
+            'name_comment' => 'Toto jméno je zobrazeno v seznamu skupin na formuláři správce.',
+            'description_field' => 'Popis',
+            'code_field' => 'Kód',
+            'code_comment' => 'Zadejte unikátní kód pro přístup přes API.',
+            'menu_label' => 'Role',
+            'list_title' => 'Správa rolí',
+            'new' => 'Nová role',
+            'delete_confirm' => 'Opravdu chcete smazat tuto roli?',
+            'return' => 'Zpět na seznam rolí',
+            'users_count' => 'Users'
```
Comment (chesio) [inline modules/backend/lang/cs/lang.php]: Again, not actually translated.

File: modules/backend/lang/cs/lang.php [hunk 3]
```diff
@@ -290,21 +404,30 @@
...
-        'primary_color' => 'Primární color',
-        'secondary_color' => 'Sekundární color',
-        'accent_color' => 'Accent color',
+        'primary_color' => 'Primární barva',
+        'secondary_color' => 'Sekundární barva',
+        'accent_color' => 'Barva pro zvýraznění',
         'styles' => 'Styly',
-        'custom_stylesheet' => 'Vlastní kaskádové styly'
+        'custom_stylesheet' => 'Vlastní kaskádové styly',
+        'navigation' => 'Navigation',
```
Comment (chesio) [inline modules/backend/lang/cs/lang.php]: Not actually translated (including the 4 following labels).

**Comment by jan-vince** [comment]
@chesio thanks for comments!

*It was a late night work and I have overlooked these (and have found some more myself).*

File: modules/cms/lang/cs/lang.php [hunk 4]
```diff
@@ -300,14 +300,14 @@
...
-        'no_files_found' => 'No files found by your request.',
+        'no_files_found' => 'Pro tento požadavek nebyly nalezeny žádné soubory.',
         'delete_empty' => 'Vyberte položky ke smazání.',
         'delete_confirm' => 'Opravu chcete smazat vybrané položky?',
         'error_renaming_file' => 'Přejmenování se nezdařilo.',
         'new_folder_title' => 'Nová složka',
         'folder_name' => 'Název složky',
         'error_creating_folder' => 'Chyba vytváření složky',
-        'folder_or_file_exist' => 'A folder or file with the specified name already exists.',
+        'folder_or_file_exist' => 'Soubor nebo soubor se zadaným názvem již existuje.',
```
Comment (chesio) [inline modules/cms/lang/cs/lang.php]: I think this should be translated as "**Adresář** nebo soubor [...]".

**Comment by chesio** [comment]
> Thanks for comments!

Well, it's always easier to review translations than to create them, so thanks! 👍

File: modules/cms/lang/cs/lang.php (hunk 4 above)
Comment (jan-vince) [inline modules/cms/lang/cs/lang.php]: Sure, I will update it. Thx!

**Comment by LukeTowers** [comment]
Thanks @jan-vince for making these translations and thanks @chesio for reviewing this!

**Pull request merged**